chef/automate
# Jamie/fix datetime compliance reports

### :nut_and_bolt: Description: What code changed, and why?
This is a continued effort to standardize time displays across the site.  This branch adds RFC2822 time format to the compliance reporting pages.  It also introduces a second time format which will be used that reads ex. `Tue, Sept 24 2019`

### :chains: Related Resources
https://github.com/chef/automate/issues/1678
https://github.com/chef/automate/issues/1679

### :+1: Definition of Done
Compliance Pages display with the new requested date format.

### :athletic_shoe: How to Build and Test the Change
1. build components/automate-ui-devproxy && start_all_services
2. Navigate to https://a2-dev.test/compliance/reports/overview
3. Click `report meta data` to view meta data --> View Report date
4. Scroll down to Chart, View Node Status Over Time Graph --> View date range under graph
5. Click on Profile Status pill to view Control Status Over Time Graph --> View date range under graph
6.  Click on `Nodes` tab, Click on any node in list to see its specific page --> View date range next to `Last Scan`
7. Click on `Scan History` in top right to view sidebar slideout --> View formatted date

### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
Reports Meta Info
<img width="895" alt="Reports - Meta Info" src="https://user-images.githubusercontent.com/16737484/66417934-ea412100-e9b5-11e9-906c-c8c7fcb44f75.png">

Node Status Graph
<img width="864" alt="Node Status Graph" src="https://user-images.githubusercontent.com/16737484/66417935-ea412100-e9b5-11e9-9af1-b5e12f1de9a2.png">

Control Status Graph
<img width="847" alt="Control Status Graph" src="https://user-images.githubusercontent.com/16737484/66417936-ea412100-e9b5-11e9-8fea-35a3c2fae3b7.png">

Reports - Specific Node and Sidebar
<img width="1119" alt="Reports - specific Node" src="https://user-images.githubusercontent.com/16737484/66417937-ead9b780-e9b5-11e9-917a-1628e7ceace3.png">
